.travis.yml

```diff
@@ -16,4 +16,4 @@ matrix:
     script:
       - make dev
     after_success:
-      - make travis_coverage
+      - travis_wait make travis_coverage
```

Comment:
em, does PD can run 20m+ in travis? If yes, it is too long, we must speed up the test.